.circleci/config.yml

```diff
@@ -0,0 +1,53 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10
+    environment:
+      CLUSTER_CTL_VERSION: 0.8.0
+    steps:
+      - checkout
+      - run:
+          name: Build the optimized site to ./public
+          command: |
+            make
+      - run:
+          name: Install ipfs-cluster-ctl binary
+          command: |
+            wget https://dist.ipfs.io/ipfs-cluster-ctl/v${CLUSTER_CTL_VERSION}/ipfs-cluster-ctl_v${CLUSTER_CTL_VERSION}_linux-amd64.tar.gz
+            tar xvfz ipfs-cluster-ctl_v${CLUSTER_CTL_VERSION}_linux-amd64.tar.gz
+            sudo mv ipfs-cluster-ctl/ipfs-cluster-ctl /usr/local/bin
+      - run:
+          name: Download dnslink script
+          command: |
+            wget https://raw.githubusercontent.com/protocol/infra-team/master/website_deploys/dnslink.sh
+            sudo chmod +x dnslink.sh
```

Comment:
we should consider using [dnslink-dnsimple](https://github.com/ipfs/dnslink-dnsimple) instead of the bash script once https://github.com/ipfs/dnslink-dnsimple/pull/8 is merged (I need to do a small fmt fix there and make a release that we could go download)

Reply:
Yes pls, that dnslink.sh script as moved or is otherwise unavailable now.